views/backend/new.php

```diff
@@ -35,15 +33,15 @@
                             <div class="card-content white-text">
                                 <?php
                                 foreach($errors as $error){
-                                    echo $error."<br/>";
+                                    echo $error;
                                 }
                                 ?>
                             </div>
                         </div>
                     <?php
                     }else{
                         $_SESSION['admin'] = $email;
-                        header("Location:index.php?page=password");
+                        header("Location:/modification-du-mot-de-passe");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The use of function header() is discouraged](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5344015)